scripts/data-retention-remove-user-data.py

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/env python
 """
 Our data retention policy is that users have accounts deactivated and personal data stripped after 3 years without
-a login.
+a login. This script not only does that but also removes any such users' email addresses from any of our mailing lists.
 
-This script is very simple and has not been upgraded to accept any arguments to prevent the possibility of accidental
-deletion. If you are in doubt use the dry run option.
+This script is simple and has deliberately not been designed without many arguments to reduce the possibility of
+accidental deletion. If you are in doubt use the dry run option.
```

Comment:
Just spotted this - I'd remove the "not" from "deliberately not designed"...

Reply:
Bloody hell